.github/workflows/tiledb-csharp.yml

```diff
@@ -151,6 +151,13 @@ jobs:
     - name: Download tiledb
       run: ./.github/scripts/download_tiledb.sh
 
+    # Download tiledb
+    - name: Copy tiledb lib
+      shell: pwsh
+      run: |
+        $ErrorView = "NormalView"
+        cp sources/TileDB.CSharp/runtimes/linux-x64/native/libtiledb.so.2.9 sources/TileDB.CSharp/runtimes/linux-x64/native/libtiledb.so
```

Comment:
This just prepares in the CI container the file to have the proper naming for the resolver once the package gets installed. Only `libtiledb.so` will get in the package, so it will not be duplicated. This also means we need the physical file not a symlink